modules/@apostrophecms/schema/ui/apos/components/AposInputString.vue

```diff
@@ -55,7 +55,7 @@ export default {
         if (this.field.type === 'float' || this.field.type === 'integer') {
           return 'number';
         }
-        if (this.field.type === 'string') {
+        if (this.field.type === 'string' || this.field.type === 'slug') {
```

Comment:
This will go into the `type` attribute on the input. There's no `type="slug"` input in HTML.